vendor/theme/include/scss/_user.scss

```diff
@@ -1,3 +1,40 @@
 /*------------------------------------
   File for your custom SCSS style
 ------------------------------------*/
+
+.select2-container--bootstrap .select2-selection--single {
+  height: 48px !important;
+  padding: 7px 5px 7px 15px !important;
+  border-radius: 0.125rem !important;
+  box-shadow: none !important;
+  border: 1px solid #d5dae2 !important;
+  font-size: 1rem !important;
+  line-height: 2 !important;
+}
+
+.select2-selection--multiple {
+  padding: 7px 5px !important;
+  font-size: 0.875rem !important;
+  line-height: 1.5 !important;
+  border-radius: 0.125rem !important;
+  box-shadow: none !important;
+  border: 1px solid #d5dae2 !important;
+  
+  .select2-selection__choice {
+    margin: 5px 0 0 6px !important;
+    padding: 0 6px !important;
+    position: relative !important;
+    display: inline-block !important;
+    font-size: .875rem !important;
+    color: #77838f !important;
+    background: rgba(119, 131, 143, 0.1) !important;
+    border-radius: 0.25rem !important;
+    border: none !important;
+    padding: .25rem .75rem !important;
+    margin: 0 0 .25rem .25rem !important;
+
+    .select2-selection__choice__remove {
+      margin-right: 6px !important;
+    }
+  }
+}
```

Comment:
I think we should move our custom CSS out of the vendor dir and under `app/javascripts/src/app/stylesheets`. Thoughts?